app/routes/_protected.send.cashu_.bolt11.$cashuSendQuoteId.tsx

```diff
@@ -0,0 +1,217 @@
+            <div className="mb-2 flex justify-center">
+              <ClockIcon className="h-16 w-16 text-orange-500" />
+            </div>
+            <div className="flex w-full flex-col gap-6">
+              <TransactionDetail label="Account" value={data.account.name} />
+              <TransactionDetail
+                label="Destination"
+                value={formattedDestination}
+              />
+              <TransactionDetail
+                label="Amount attempted"
+                value={data.amountToReceive.toLocaleString({
+                  unit: getDefaultUnit(data.amountToReceive.currency),
+                })}
+              />
+            </div>
+          </div>
+        </PageContent>
+        <PageFooter className="pb-14">
+          <Button asChild className="w-[80px]">
+            <LinkWithViewTransition
+              to="/send"
+              transition="slideDown"
+              applyTo="oldView"
+            >
+              Ok
+            </LinkWithViewTransition>
+          </Button>
+        </PageFooter>
+      </Page>
+    );
+  }
+
+  if (data?.state === 'FAILED') {
+    return (
+      <Page>
+        <PageHeader>
+          <PageHeaderTitle>Payment Failed</PageHeaderTitle>
+          <ClosePageButton to="/" transition="slideDown" applyTo="oldView" />
+        </PageHeader>
+        <PageContent className="relative flex flex-col items-center gap-12">
+          <MoneyWithConvertedAmount money={data.amountToReceive} />
+          <div className="fade-in-0 flex w-full animate-in flex-col gap-12 place-self-center duration-300">
+            <div className="mb-2 flex justify-center">
+              <AlertCircle className="h-16 w-16 text-red-500" />
+            </div>
+            <div className="flex w-full flex-col gap-6">
+              <TransactionDetail label="Account" value={data.account.name} />
+              <TransactionDetail
+                label="Destination"
+                value={formattedDestination}
+              />
+              <TransactionDetail
+                label="Amount attempted"
+                value={data.amountToReceive.toLocaleString({
+                  unit: getDefaultUnit(data.amountToReceive.currency),
+                })}
+              />
+              {data.failureReason && (
+                <TransactionDetail label="Reason" value={data.failureReason} />
+              )}
+            </div>
+          </div>
+        </PageContent>
+        <PageFooter className="pb-14">
+          <Button asChild className="w-[80px]">
+            <LinkWithViewTransition
+              to="/send"
+              transition="slideDown"
+              applyTo="oldView"
+            >
+              Ok
+            </LinkWithViewTransition>
+          </Button>
+        </PageFooter>
+      </Page>
+    );
+  }
+
+  // vvvv TODO: remove this below and handle the other states where needed vvvvv
```

Comment:
is this for unpaid state? when would this happen? should we just do some redirect instead or throw error instead of building ui?